packages/analytics-js-integrations/src/integrations/Braze/browser.js

```diff
@@ -37,11 +37,24 @@ class Braze {
 
     if (config.dataCenter) {
       // ref: https://www.braze.com/docs/user_guide/administrative/access_braze/braze_instances
-      const dataCenterArr = config.dataCenter.trim().split('-');
-      if (dataCenterArr[0].toLowerCase() === 'eu') {
-        this.endPoint = `sdk.fra-${dataCenterArr[1]}.braze.eu`;
-      } else {
-        this.endPoint = `sdk.iad-${dataCenterArr[1]}.braze.com`;
+      const [dataCenterRegion, dataCenterNumber] = config.dataCenter
```

Comment:
As this is older integration, test cases are missing - let's add with at least covering new code